### PR TITLE
LightProbe: Moved utility methods to examples.

### PR DESCRIPTION
**Pull request by mrdoob**
See #16271

### CODE REVIEW

File: examples/webgl_lightprobe.html [hunk 1]
```diff
@@ -125,8 +126,8 @@
 
 					scene.background = cubeTexture;
 
-					lightProbe.setFromCubeTexture( cubeTexture );
-
+					lightProbe.copy( THREE.LightProbeGenerator.fromCubeTexture( cubeTexture ) );
```
Comment (WestLangley) [inline examples/webgl_lightprobe.html]: probably should be
```js
lightProbe = THREE.LightProbeGenerator.fromCubeTexture( cubeTexture );
```
but that requires a refactoring of the example.

Comment (mrdoob) [reply examples/webgl_lightprobe.html]: Yeah. Seemed like the simplest solution and it was not too bad 😇

File: examples/js/lights/LightProbeGenerator.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
+/**
+ * @author WestLangley / http://github.com/WestLangley
+ */
+
+THREE.LightProbeGenerator = {
+
+	fromAmbientLight: function ( light ) {
```
Comment (WestLangley) [inline examples/js/lights/LightProbeGenerator.js]: I am not a fan of this API, to be honest. What is the workflow where one would instantiate an ambient light (or hemisphere light) and then use it to instantiate a light probe?

Comment (mrdoob) [reply examples/js/lights/LightProbeGenerator.js]: I think it's just temporal. Something like `new THREE.AmbientLightProbe()` sounds better to me.

Comment (WestLangley) [reply examples/js/lights/LightProbeGenerator.js]: That was actually the direction I was heading...

I do not have a problem with moving `.fromCubeTexture()` to the examples, but I prefer #16298.

I think we have time to fix this before the next release.

**Pull request merged**